includes/classes/admin/modules/class-tailwind.php

```diff
@@ -906,6 +915,56 @@ public function get_prefix() {
             return apply_filters( 'pip/tailwind/config/prefix', pip_maybe_get( $configuration, 'prefix' ) );
         }
 
+        /**
+         * Load TailwindCSS CDN
+         */
+        public function load_tailwind_cdn() {
+
+            // Load TailwindCSS CDN only once
+            if ( wp_script_is( 'tailwind-cdn' ) ) {
+                return;
+            }
+
+            // Load TailwindCSS CDN
+            $tailwind_version = apply_filters( 'pip/tailwind/cdn_version', '' );
+            wp_enqueue_script( 'tailwind-cdn', "https://cdn.tailwindcss.com/$tailwind_version", array(), $tailwind_version, false );
+
+            // Load Pilo'Press Tailwind config on front
+            $tailwind_config = pip_get_tailwind_config();
+            if ( $tailwind_config ) {
+
+                // Replace data to match Tailwind CDN config
+                $tailwind_config = str_replace( 'module.exports', 'tailwind.config', $tailwind_config );
+                wp_add_inline_script( 'tailwind-cdn', $tailwind_config );
+
+            }
+        }
+
+        /**
+         * Load Pilo'Press custom style for the CDN
+         */
+        public function tailwind_cdn_config() {
+
+            // Load Pilo'Press Tailwind style on front
+            $tailwind_css = pip_get_tailwind_css() . pip_get_layouts_css();
```

Comment:
To be extra safe, i would add an empty space there:
 `$tailwind_css = pip_get_tailwind_css() . ' ' . pip_get_layouts_css();`